src/Sylius/Bundle/AdminBundle/Resources/views/Taxon/Macro/tree.html.twig

```diff
@@ -0,0 +1,25 @@
+{% macro render(taxons) %}
+    <div class="ui list">
+        {{ _self.renderLeaf(taxons) }}
+    </div>
+{% endmacro %}
+
+{% macro renderLeaf(taxons) %}
+    {% import '@SyliusUi/Macro/buttons.html.twig' as buttons %}
+    {% for taxon in taxons %}
+        <div class="item">
+            <i class="folder icon"></i>
+            <div class="content">
+                <div class="header">{{ taxon.name }}</div>
+                {{ buttons.delete(path('sylius_admin_taxon_delete', { 'id': taxon.id }), 'sylius.ui.delete'|trans) }}
+                {{ buttons.edit(path('sylius_admin_taxon_update', { 'id': taxon.id })) }}
+                {% if not taxon.description %}
+                    <div class="description">{{ taxon.description }}</div>
```

Comment:
This should be removed anyway, description can be too long to display in the tree.